src/client.ts

```diff
@@ -17,6 +17,7 @@
  */
 
 import {net, hrTime} from "./adapter.node";
+import * as tls from "tls";
```

Comment:
An export for `tls` should be added to `adapter.node` and then imported from there. Currently it looks like Deno doesn't support ALPN, so I think the only thing we can do is export some mocked tls api from `adapter.deno` that falls back to the plaintext protocol?

Reply:
I'm under the impression that plane text is not an option going forward... @1st1 what do you think about this?

Reply:
changed